docs/.vale/styles/Quarkus/Spelling.yml

```diff
@@ -279,7 +285,6 @@ filters:
   - Minishift
   - Mirantis
   - Mockito
-  - MongoDB
```

Comment:
Not sure why this was removed. Did the tooling do that automatically?

Reply:
It wasn't removed. It was replaced by a regex that handles lowercase m and capital d too.